src/power_grid_model_io/converters/pandapower_converter.py

```diff
@@ -1240,6 +1215,55 @@ def _generate_ids(self, pp_table: str, pp_idx: pd.Index, name: Optional[str] = N
         self.next_idx += n_objects
         return pgm_idx
 
+    def _pp_ward_output(self):
```

Comment:
You create this new function. But where did you call the function in the main output conversion function?